cfme/networks/topology.py

```diff
@@ -0,0 +1,206 @@
+from widgetastic.browser import Browser
+from navmazing import NavigateToAttribute
+
+import re
+import time
+from cfme.networks.topology_view import TopologyView
+from utils.appliance import Navigatable
+from utils.appliance.implementations.ui import CFMENavigateStep, navigator, navigate_to
+from utils.browser import manager
+from wait_for import wait_for
+
+
+class Topology(Navigatable):
+    LEGENDS = '//kubernetes-topology-icon'
+    ELEMENTS = '//kubernetes-topology-graph//*[name()="g"]'
+    LINES = '//kubernetes-topology-graph//*[name()="lines_obj"]'
+
+    browser = None
+
+    def __init__(self, appliance):
+        Navigatable.__init__(self, appliance=appliance)
+        self.legends_obj = []
+        self.elements_obj = []
+        self.lines_obj = []
+        self.element_ref = None
+        self.display_names = None
+        self.view = None
+
+        url_key = self.appliance.server.address()
+        Topology.browser = Browser(manager.ensure_open(url_key))
```

Comment:
You should not do this, this is "managing" browsers outside of browser manager scope.

Reply:
Is there a better way of getting widgetastic browser object?